README.md

```diff
@@ -84,7 +84,9 @@ Clone this repository to your desired folder:
 
 ## 🔭 Future Features <a name="future-features"></a>
 
-- [ ] **Pending**
+- [ ] **Home page**
+- [ ] **Drag and Drop**
+- [ ] **contact page**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
- [ ] As suggested by the previous reviewer , kindly add a link to the contribution section, a I noticed you didn't make any comits . 

## Exected :
![image](https://user-images.githubusercontent.com/50990925/222057715-70e1fb85-fa18-499c-83e9-027e2d66c034.png)

Reply:
@Shedrack-Sunday 
dear I have added the link below contribution section and committed that kindly check everything